### PR TITLE
8332251: javadoc: incorrect method references in Region and PopupControl

### PR DESCRIPTION
**Pull request by andy-goryachev-oracle**
The javadoc for `Region.getPrefHeight() / getPrefWidth()` incorrectly refers to `getPrefHeight(forWidth) / getPrefWidth(forHeight)`

should be

`prefHeight(forWidth) / prefWidth(forHeight)`

- same issue is also fixed in `PopupControl`.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8332251](https://bugs.openjdk.org/browse/JDK-8332251): javadoc: incorrect method references in Region and PopupControl (**Bug** - P4)

### Reviewers
 * [Ambarish Rapte](https://openjdk.org/census#arapte) (@arapte - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx.git pull/1456/head:pull/1456` \
`$ git checkout pull/1456`

Update a local copy of the PR: \
`$ git checkout pull/1456` \
`$ git pull https://git.openjdk.org/jfx.git pull/1456/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1456`

View PR using the GUI difftool: \
`$ git pr show -t 1456`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx/pull/1456.diff">https://git.openjdk.org/jfx/pull/1456.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jfx/pull/1456#issuecomment-2113623650)

### CODE REVIEW

File: modules/javafx.graphics/src/main/java/javafx/scene/layout/Region.java [hunk 1]
```diff
@@ -1209,7 +1209,7 @@ public final DoubleProperty prefWidthProperty() {
      * doesn't meet the application's layout needs.
      * <p>
      * Defaults to the <code>USE_COMPUTED_SIZE</code> flag, which means that
-     * <code>getPrefHeight(forWidth)</code> will return the region's internally
+     * {@link #prefHeight(forWidth)} will return the region's internally
```
[the remark this answers is not in the document]
Comment (andy-goryachev-oracle) [reply modules/javafx.graphics/src/main/java/javafx/scene/layout/Region.java:1212]: thank you for pointing this out, I am going to revert back to `<code>` and update `PopupControl`.

[the remark this answers is not in the document]
Comment (arapte) [reply modules/javafx.graphics/src/main/java/javafx/scene/layout/Region.java]: Looks good,
I recommend to update the summary, current summary is quite specific to Region / prefWidth/Height.

A suggestion->
javadoc: incorrect method references in Region and PopupControl

**Review by arapte — APPROVED** [review]

File: modules/javafx.graphics/src/main/java/javafx/scene/layout/Region.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (andy-goryachev-oracle) [reply modules/javafx.graphics/src/main/java/javafx/scene/layout/Region.java]: excellent suggestion! done.

**Pull request closed without merging**